src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-listeners-form/nvmeof-listeners-form.component.html

```diff
@@ -0,0 +1,77 @@
+<cd-modal [pageURL]="pageURL"
+          [modalRef]="activeModal">
+  <span class="modal-title"
+        i18n>{{ action | titlecase }} {{ resource | upperFirst }}</span>
+  <ng-container class="modal-content">
+    <form name="listenerForm"
+          #formDir="ngForm"
+          [formGroup]="listenerForm"
+          novalidate>
+      <div class="modal-body">
+        <!-- Host -->
+        <div class="form-group row">
+          <label class="cd-col-form-label"
+                 for="host">
+            <span class="required"
+                  i18n>Host Name</span>
+          </label>
+          <div class="cd-col-form-input">
+            <select id="host"
+                    name="host"
+                    class="form-select"
+                    formControlName="host">
+              <option *ngIf="hosts === null"
+                      [ngValue]="null"
+                      i18n>Loading...</option>
+              <option *ngIf="hosts && hosts.length === 0"
+                      [ngValue]="null"
+                      i18n>-- No hosts available --</option>
+              <option *ngIf="hosts && hosts.length > 0"
+                      [ngValue]="null"
+                      i18n>-- Select a host --</option>
+              <option *ngFor="let h of hosts"
```

Comment:
btw, I've been meaning to point this out early. but let's have meaningful variables.
```suggestion
              <option *ngFor="let host of hosts"
```

Reply:
`host` m using above as control name hence "h", so do not want to confuse while reading